tests/app/db.py

```diff
@@ -96,6 +97,20 @@ def create_user(
     return user
 
 
+def get_unique_phone_number_for_user():
```

Comment:
This could still be useful for testing, nice!  I wonder if we should change the area code to also be `555` just to be sure).

[There's more info here](https://en.wikipedia.org/wiki/555_(telephone_number)), and it looks like the official range of fake numbers is `555-0100` through `555-0199`.